httprequest_lego_provider/serializers.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""Serializers."""
+
+from rest_framework import serializers
```

Comment:
Nitpicky comment, but would renaming the module to something like `domain_serializer.py` help remove conflicting namespace for `serializers`?

Reply:
What conflicting namespace are you talking about?

Reply:
for example, when importing `serializers` module from lets say `charm.py`, `from rest_framework import serializers` this can no longer be imported with the same name.

Reply:
Sure!